pkg/setting/types.go

```diff
@@ -28,3 +28,64 @@ const (
 	// MinRequest 2 CPU 2 G
 	MinRequest Request = "min"
 )
+
+const (
+	Aslan     = iota + 1 // 1
```

Comment:
I think it is more appropriate to use string constants here

Reply:
string is better for user interface, but here, I think int is better